docs/src/pages/docs/getting-started/app-router/with-i18n-routing.mdx

```diff
@@ -220,17 +220,15 @@ const withNextIntl = createNextIntlPlugin(
 The `locale` that was matched by the middleware is available via the `locale` param and can be used to configure the document language. Additionally, we can use this place to pass configuration from `i18n/request.ts` to Client Components via `NextIntlClientProvider`.
 
 ```tsx filename="app/[locale]/layout.tsx"
-import {NextIntlClientProvider, Locale, hasLocale} from 'next-intl';
+import {NextIntlClientProvider, hasLocale} from 'next-intl';
 import {notFound} from 'next/navigation';
 import {routing} from '@/i18n/routing';
+import type { PropsWithChildren } from 'react';
```

Comment:
This might be a matter of personal taste, I'd go with the lead from the Next.js docs here which [type `children` explicitly](https://nextjs.org/docs/app/getting-started/layouts-and-pages#creating-a-layout).

```suggestion
```

Reply:
Sure!